app/snapshot.go

```diff
@@ -167,14 +168,7 @@ func rmSnapshot(c *cli.Context) error {
 	return lastErr
 }
 
-func lsSnapshot(c *cli.Context) error {
-	cli := getCli(c)
-
-	replicas, err := cli.ListReplicas()
-	if err != nil {
-		return err
-	}
-
+func filterSnapshot(replicas []rest.Replica) ([]string, error) {
```

Comment:
fn name doesn't suit

Reply:
we are filtering the snapshot which is common in all the replicas.